pyproject.toml

```diff
@@ -23,7 +23,7 @@ dependencies = [
     "pydantic>=2.7,<2.9",
     "pymongo==4.3.3",
     "dask==2023.5.0",
-    "aind-data-schema==1.0.0",
+    "aind-data-schema==1.2.0",
```

Comment:
Should I update this to just a lower bound?

Reply:
Nah, we can keep it like this since this is a standalone app. Can we update pydantic to the latest version? Some of the slow down I've noticed with the tests and aind-metadata-service can be alleviated if we upgrade to the latest version of pydantic.